lib/slack/web/api/errors/slack_error.rb

```diff
@@ -22,6 +22,12 @@ def errors
           def response_metadata
             response.body.response_metadata
           end
+
+          def to_s
```

Comment:
the approach I took here is to put as many details as possible but without empty details i.e. the errors list would not be added if there are none, same as the response metadata.